ert3/config/_parameters_config.py

```diff
@@ -185,14 +188,8 @@ class _ParameterConfig(_ParametersConfig):
 
     @root_validator
     def _ensure_variables_or_size(cls, values: Dict[str, Any]) -> Dict[str, Any]:
-        has_variables = values.get("variables")
-        has_size = values.get("size")
-        if has_variables and has_size:
+        if values.get("variables") and values.get("size"):
             raise AssertionError("Parameter group cannot have both variables and size")
```

Comment:
This is more like a `ValueError`. But did you intend the change in semantics here?

Reply:
Agree, changed to `ValueError`.

But yes, scalar records are obtained by doing exactly what was not allowed before.